Orange-OpenSource/Orange-Boosted-Bootstrap
# 🚀 Release 5.1.3

⚠️ Don't merge this PR! It must not go into `main` branch. This is a patch coming from Bootstrap.

- [x] `git checkout -b v513`
- [x] Merge the 2 commits in https://github.com/twbs/bootstrap/commits/v5.1.3
- [x] Delete this branch and close this PR

---

  - Firefox ESR
  - IE11 (v4 only)
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - check wrong matches in `CHANGELOG.md`, and maybe `site/content/docs/<version>/migration.md`
  - :warning: check the `package-lock.json` and `package.json` content, only "boosted" should have its version changed!
  - manually update the version in `nuget/boosted.nuspec` and `nuget/boosted.sass.nuspec`
  - :warning: `site/content/docs/5.1/**/*.md` should not always be modified
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - and probably maintain [a ship list (eg for v4.4.0)](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/226)
- ~~merge (on `v4-dev` or `main`)~~
  - attach zip file
  - paste CHANGELOG / Ship list in the release's description
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
  - [x] copy `../_site` (main) or `../bs-docs` (v4-dev) to the `gh-pages` branch
